### PR TITLE
Feature/implement social sign up layout

### PR DESCRIPTION
**Pull request by benikgalstyan**

### CODE REVIEW

File: lib/presentation/pages/social_sign_up_screen/social_sign_up_layout.dart [hunk 1]
```diff
@@ -0,0 +1,70 @@
...
+  Widget build(BuildContext context) {
+    return Scaffold(
+      body: BackgroundWidget(
+        child: Center(
+          child: Column(
+            children: [
+              const Spacer(flex: upperFlex),
+              appLogo,
+              Text(
+                context.s.askm,
```
Comment (k1mka) [inline lib/presentation/pages/social_sign_up_screen/social_sign_up_layout.dart]: shame, put it in a separate widget like AppLogo

File: lib/presentation/pages/social_sign_up_screen/social_sign_up_layout.dart [hunk 2]
```diff
@@ -0,0 +1,70 @@
...
+class SocialSignUpLayout extends StatefulWidget {
+  const SocialSignUpLayout({super.key});
+
+  @override
+  State<SocialSignUpLayout> createState() => _SocialSignUpLayoutState();
+}
+
+class _SocialSignUpLayoutState extends State<SocialSignUpLayout> {
+  static const upperFlex = 5;
+  final appLogo = SvgPicture.asset(Assets.images.logo);
```
Comment (k1mka) [inline lib/presentation/pages/social_sign_up_screen/social_sign_up_layout.dart]: You're cluttering up the layout with such things, there's no need for them in the layout, it's better to store them in the widget

File: lib/presentation/pages/social_sign_up_screen/social_sign_up_layout.dart [hunk 3]
```diff
@@ -0,0 +1,70 @@
...
+                context.s.askm,
+                style: const TextStyle(fontSize: Spacings.xxL),
+              ),
+              Text(
+                context.s.whether,
+                style: const TextStyle(fontSize: Spacings.L),
+              ),
+              Text(
+                context.s.comfort,
+                style: const TextStyle(fontSize: Spacings.L),
```
Comment (k1mka) [inline lib/presentation/pages/social_sign_up_screen/social_sign_up_layout.dart]: create an application in the core theme/palette/light_palette and textStyles 

And take out all the styles for each component separately, and use colors from there too

File: lib/presentation/pages/social_sign_up_screen/social_sign_up_layout.dart [hunk 4]
```diff
@@ -0,0 +1,70 @@
...
+              const SizedBox(
+                height: Spacings.L,
+              ),
+              CustomButton(
+                onPressed: () {
+                  // TODO(Benik): implement email sign up
+                },
+                textButtonColor: Colors.white,
+                text: context.s.continueWithEmail,
+                buttonColor: const Color(0xFF292727),
```
Comment (k1mka) [inline lib/presentation/pages/social_sign_up_screen/social_sign_up_layout.dart]: from light palette

File: lib/presentation/pages/social_sign_up_screen/social_sign_up_layout.dart [hunk 5]
```diff
@@ -0,0 +1,70 @@
...
+                text: context.s.continueWithGoogle,
+              ),
+              const SizedBox(
+                height: Spacings.L,
+              ),
+              CustomButton(
+                onPressed: () {
+                  // TODO(Benik): implement email sign up
+                },
+                textButtonColor: Colors.white,
```
Comment (k1mka) [inline lib/presentation/pages/social_sign_up_screen/social_sign_up_layout.dart]: from light palette

File: lib/presentation/widgets/background_widget.dart [hunk 6]
```diff
@@ -0,0 +1,25 @@
...
+  final Widget child;
+
+  @override
+  Widget build(BuildContext context) {
+    return Stack(
+      children: [
+        Image.asset(
+          Assets.images.backgroundImage.keyName,
+          fit: BoxFit.cover,
+          width: double.infinity,
```
Comment (k1mka) [inline lib/presentation/widgets/background_widget.dart:20]: are you shoure whay you need to write wigth and height?

File: lib/presentation/widgets/custom_button.dart [hunk 7]
```diff
@@ -0,0 +1,75 @@
+import 'package:askm/presentation/tokens/spacing.dart';
+import 'package:flutter/material.dart';
+
+class CustomButton extends StatelessWidget {
```
Comment (k1mka) [inline lib/presentation/widgets/custom_button.dart]: i dont like this widget with a lot of fields, you need two type of button, how i was said before, also add prefix app name like ASKMElevatedButoon and ASKMOutlinedButoon

File: lib/presentation/widgets/background_widget.dart (hunk 6 above)
Comment (benikgalstyan) [reply lib/presentation/widgets/background_widget.dart:20]: Yes i am

File: lib/core/theme/text_styles.dart [hunk 8]
```diff
@@ -0,0 +1,20 @@
...
+class TextStyles {
+  static const TextStyle headline1 = TextStyle(
+    fontSize: Spacings.xxL,
+    color: LightPalette.whiteColor,
+  );
+
+  static const TextStyle bodyText1 = TextStyle(
+    fontSize: Spacings.L,
+    color: LightPalette.whiteColor,
+  );
```
Comment (k1mka) [inline lib/core/theme/text_styles.dart:17]: spacer?

**Pull request merged**